core/iwasm/aot/aot_reloc.h

```diff
@@ -37,19 +37,6 @@ typedef struct {
 #define REG_AOT_TRACE_SYM()
 #endif
 
-#if (defined(_WIN32) || defined(_WIN32_)) && defined(NDEBUG)
-#define REG_COMMON_SYMBOLS                \
-    REG_SYM(aot_set_exception_with_id),   \
-    REG_SYM(aot_invoke_native),           \
-    REG_SYM(aot_call_indirect),           \
-    REG_SYM(wasm_runtime_enlarge_memory), \
-    REG_SYM(wasm_runtime_set_exception),  \
-    {"memset", (void*)aot_memset},        \
-    {"memmove", (void*)aot_memmove},      \
-    REG_BULK_MEMORY_SYM()                 \
-    REG_ATOMIC_WAIT_SYM()                 \
-    REG_AOT_TRACE_SYM()
-#else /* else of (defined(_WIN32) || defined(_WIN32_)) && defined(NDEBUG) */
```

Comment:
some symbols are not exists in Win64 Release build, maybe we need to reserve these code but change `WIN32` to `WIN64`

``` C
#if (defined(_WIN64) || defined(_WIN64_)) && defined(NDEBUG)
#define REG_COMMON_SYMBOLS                \
REG_SYM(aot_set_exception_with_id),       \
    REG_SYM(aot_invoke_native),           \
    REG_SYM(aot_call_indirect),           \
    REG_SYM(wasm_runtime_enlarge_memory), \
    REG_SYM(wasm_runtime_set_exception),  \
    {"memset", (void*)aot_memset},        \
    {"memmove", (void*)aot_memmove},      \
    REG_SYM(fmin),                        \
    REG_SYM(fminf),                       \
    REG_SYM(fmax),                        \
    REG_SYM(fmaxf),                       \
    REG_SYM(trunc),                       \
    REG_SYM(truncf),                      \
    REG_SYM(rint),                        \
    REG_SYM(rintf),                       \
    REG_BULK_MEMORY_SYM()                 \
    REG_ATOMIC_WAIT_SYM()                 \
    REG_AOT_TRACE_SYM()
#else /* else of (defined(_WIN64) || defined(_WIN64_)) && defined(NDEBUG) */
...
```

Reply:
Oh interesting. I'll try the win64 build here. I'd rather get Win64 building properly than have them removed.

Reply:
I pushed an update with x64 AoT relocs in place. I totally forgot about that, my bad!